### DISCUSSION TITLE
How do I make the blog posts wider?

### DISCUSSION THREAD

**Discussion by 0xGodspeed** [chrismwilliams/astro-theme-cactus#141] [Q&A]
I write a lot of code and a lot of times it doesn't fit.
How do I make the posts wider?
I tried fiddling around with the CSS and flex stuff but couldn't figure it out.

**Comment by chrismwilliams** [comment]
There's a couple of places you can try. The theme uses tailwind typography for styling markdown and they [set a prose width on content](https://tailwindcss.com/docs/typography-plugin#overriding-max-width). Just enter a new value [here](https://github.com/chrismwilliams/astro-theme-cactus/blob/5f248aafb975f392243474040024bc8657a129e5/src/layouts/BlogPost.astro#L47).

If that's not enough then the [body tag has a max width](https://github.com/chrismwilliams/astro-theme-cactus/blob/5f248aafb975f392243474040024bc8657a129e5/src/styles/global.css#L32) of 3xl so you could increase that to suit. I can't guarantee however things not breaking style wise using this method.

Hope that helps
